curriculum/challenges/portuguese/01-responsive-web-design/css-flexbox/apply-the-flex-direction-property-to-create-a-column-in-the-tweet-embed.portuguese.md

```diff
@@ -7,10 +7,10 @@ localeTitle: Aplicar a propriedade flex-direction para criar uma coluna no Tweet
 ---
 
 ## Description
-<section id="description"> O <code>header</code> e <code>footer</code> incorporados pelo tweet usavam a propriedade <code>flex-direction</code> anteriormente com um valor de linha. Da mesma forma, os itens dentro do elemento <code>.profile-name</code> funcionariam bem empilhados como uma coluna. </section>
+<section id="description"> O <code>header</code> e <code>footer</code> incorporados pelo tweet usavam a propriedade <code>flex-direction</code> anteriormente com um valor de *row*. Da mesma forma, os itens dentro do elemento <code>.profile-name</code> ficariam bem empilhados como uma coluna. </section>
```

Comment:
@tchaffee 
The markup was added to highlight that the value that should be used (row) is in foreign language (in relation to the Portuguese).
I know the markup does not exist in the original, and that's because the need to use italics only came up with the translation.
I ask you to consider the information in this comment. However, if you still judge my solution incorrect, let me know and I will remove the markup.